functorch/functorch/_src/fx_minifier.py

```diff
@@ -296,4 +296,5 @@ def try_granularity(failing_state, granularity, use_non_granular):
     print(f"Made {num_queries} queries")
     failing_fx = fx.GraphModule(fail_f, failing_state.graph)
     dump_state(failing_fx, failing_state.inps)
+    print("Wrote minimal repro out to repro.py")
```

Comment:
raw print?

Reply:
No, this is intended - just a small QOL improvement to minifier I'm folding into this change lol.